programs/ssl/ssl_server2.c

```diff
@@ -1212,7 +1212,7 @@ int main( int argc, char *argv[] )
     const char *alpn_list[ALPN_LIST_SIZE];
 #endif
 #if defined(MBEDTLS_MEMORY_BUFFER_ALLOC_C)
-    unsigned char alloc_buf[100000];
+    unsigned char alloc_buf[120000];
```

Comment:
While at it, we might introduce this as a macro-defined constant (similar to `ALPN_LIST_SIZE`), allowing future adaptions without looking into the actual source.

Reply:
Done. Please re-review.